perfmetrics/scripts/continuous_test/ml_tests/build_dino.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
+
+echo "Setting up the machine with Docker and Nvidia Driver"
+chmod +x ml_tests/setup_host.sh
+source ml_tests/setup_host.sh
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
+echo "Building docker image containing all pytorch libraries..."
+sudo docker build . -f perfmetrics/scripts/ml_tests/pytorch_dino_model/Dockerfile --tag pytorch-gcsfuse
```

Comment:
is this docker file specific to dino_model or pytroch?

either ways, create a pytorch folder and then dino folder inside it.  same for build_dino.sh also.